test_scripts/Protocol/NACKReason/005_Wrong_app_type_Video_Audio_Services_NACK.lua

```diff
@@ -0,0 +1,61 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0308-protocol-nak-reason.md
+
```

Comment:
```suggestion
--
```
I would propose to comment out this line